wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -600,6 +602,12 @@ private void check() {
 
                 log.info("starting peergroup");
                 peerGroup = new PeerGroup(Constants.NETWORK_PARAMETERS, blockChain, headerChain);
+                if (resetMNListsOnPeerGroupStart) {
+                    resetMNListsOnPeerGroupStart = false;
+                    application.getWallet().getContext().masternodeListManager.setBootstrap(mnlistinfoBootStrapStream, qrinfoBootStrapStream, SimplifiedMasternodeListManager.QUORUM_ROTATION_FORMAT_VERSION);
+                    resetMNLists(true);
```

Comment:
`resetMNLists(true)` will remove all existing masternodes/quorums and then reload them from the bootstrap file.  After this the app can sync these lists from the point that the bootstrap was made, rather than download all data from the network.

Reply:
`resetMNLists(false)` will remove all existing masternodes/quorums and then initialize the system in an empty state.